com.unity.ml-agents.extensions/Documentation~/com.unity.ml-agents.extensions.md

```diff
@@ -18,8 +18,7 @@ The following table describes the package folder structure:
 | _Runtime_        | Contains core C# APIs for integrating ML-Agents into your Unity scene. |
 | _Tests_          | Contains the unit tests for the package.                               |
 
-The Runtime directory currently contains three features:
```

Comment:
The count was wrong before :) It would be correct now, but made it more generic.

Reply:
I made the exact same change in my PR :)